config/application.rb

```diff
@@ -39,13 +39,13 @@ class Application < Rails::Application
 
     config.middleware.insert_after Rails::Rack::Logger, Rack::Cors do
       allow do
-        origins /.*\.prx\.(?:org|dev)$/
+        origins /.*\.prx\.(?:org|dev|tech)$/
         resource '/api/*', methods: [:get, :put, :post, :options], headers: :any
       end
 
       allow do
         origins '*'
-        resource '/api/*', methods: [:get]
+        resource '/api/*', methods: [:get], headers: :any
```

Comment:
For now, yeah. I mean, there's not really a way for anyone else to get tokens at this point anyway, so it's sort of moot. Just don't want to open this up until it's necessary.